### ISSUE TITLE
make complains about -q

### ISSUE THREAD

**Issue by tommyjcarpenter** [barrybingo/exfat-nofuse#3]
When I run `make` and `make install` (since arch isn't pointed to this yet), I get the warning:

```
depmod: WARNING: Ignored deprecated option -q
```

Just an fyi

**Comment by barrybingo** [comment]
Good news is that the Arch AUR now points to this repo.
https://aur.archlinux.org/packages/exfat-dkms-git/

As to the -q option it seems this was deprecated on the tools inception

https://git.kernel.org/pub/scm/utils/kernel/kmod/kmod.git/commit/?id=64b8b586ebcc8bea7085a0a44548dff327f4dff2

The option asks for quiet yet produces a loud WARNING message :)

Yet here https://wiki.debian.org/depmod the -q is specifically mentioned but in the debian man page the -q is not mentioned and ultimately debian's source is the kernel.org i.e. https://salsa.debian.org/md/kmod/commit/64b8b586ebcc8bea7085a0a44548dff327f4dff2 

In other words.  -q has never done anything even though some people think it does

Maybe some system has it implemented so seeing as it ain't broke we should just leave it alone.

**Comment by tommyjcarpenter** [comment]
great!
